src/services/codefixes/convertToEs6Module.ts

```diff
@@ -200,7 +223,7 @@ namespace ts.codefix {
                 changes.delete(sourceFile, assignment.parent);
             }
             else {
-                const replacement = isObjectLiteralExpression(right) ? tryChangeModuleExportsObject(right)
+                const replacement = isObjectLiteralExpression(right) ? tryChangeModuleExportsObject(right, useSitesToUnqualify)

```

Comment:
I don't know this code at all, but it seems like there are a lot of potential places that need to be threaded through with `useSitesToUnqualify`. Are you confident you didn't miss any?

Reply:
Yep, pretty confident. I had never seen this code before either, but I ended up reading pretty much all the parts that actually perform text changes.